### ISSUE TITLE
Progress dialog visual issues

### ISSUE THREAD

**Issue by oskretc** [fman-community/fman#444]
Great work implementing the progress dialog.

One little thing, the dialog button seem to have some issues (see attached pics below) the 2 pics where taken using different resolutions. 

Win 10

Cheers

<img width="184" alt="capture1" src="https://user-images.githubusercontent.com/4963991/42594488-8f9a0454-854f-11e8-8f45-9b8af80dbc1b.PNG">
<img width="184" alt="capture2" src="https://user-images.githubusercontent.com/4963991/42594489-8fbaf7e0-854f-11e8-9563-98695f8d6bef.PNG">

**Comment by mherrmann** [comment]
Does it improve when you change


```
ProgressDialog {
	min-width: 25em;
	max-width: 25em;
}
```

at the end of `C:\Users\USERNAME\AppData\Local\fman\Versions\1.1.3\styles.qss` to

```
ProgressDialog {
	min-width: 120ex;
	max-width: 120ex;
}
```

? (Please test with both resolutions, if possible)

**Comment by oskretc** [comment]
Same results, no improvements with both resolutions :(

**Comment by mherrmann** [comment]
Pity, I see. Thanks. I'll keep tinkering.

**Comment by mherrmann** [comment]
I'm sorry, I am unable to reproduce - and thus fix - this. I just tested on another machine (Win 10, high DPI) and neither changing the resolution nor Windows' font size exhibited the problem.

Which resolutions did you try? Is this on a high-DPI display?

**Comment by mherrmann** [comment]
Should be fixed in [fman 1.2.6](https://fman.io/changelog#v1.2.6).
